AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_PhysicalSkyAdded.py

```diff
@@ -130,42 +192,188 @@ def AtomEditorComponents_PhysicalSky_AddedToEntity():
+
+        # 7. Set Sun Intensity value to 0 then back to 8.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Intensity'), 0)
+        Report.result(
+            Tests.sun_intensity_set_to_0,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Intensity')) == 0)
+
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Intensity'), 8.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.sun_intensity_set_to_8,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Intensity')) == 8.0)
+
+        # 8. Set Turbidity value to 5 then back to 1.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Turbidity'), 0.0)
+        Report.result(
+            Tests.turbidity_set_to_5,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Turbidity')) == 0.0)
+
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Turbidity'), 1)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.turbidity_set_to_1,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Radius Factor')) == 1)
+
+        # 9. Set Sun Radius Factor from 0.0 back to 1.0.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Radius Factor'), 0.0)
+        Report.result(
+            Tests.sun_radius_factor_set_to_0,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Radius Factor')) == 0.0)
+
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Radius Factor'), 1.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.sun_radius_factor_set_to_1,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Radius Factor')) == 1.0)
+
+        # 10a. Enable Fog for subsequent tests.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Enable Fog'), True)
+        Report.result(
+            Tests.enable_fog_set_to_true,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Enable Fog')) is True)
+        general.idle_wait_frames(1)
+
+        # 11. Change fog color to green, then return to default.
+        color_green = math.Color(0.0, 255.0, 0.0, 255.0)
+        color_default = math.Color(0.0, 0.0, 0.0, 255.0)
```

Comment:
does fog expect Vector3 rather than math.Color? the atom_constants docstring say Vector3.

Reply:
Was wrong in Atom Constants, tried vector 3 and it didn't work, had to switch to color. Fixed the comment in Atom_Constant to match.